stage1/stage1.mk

```diff
@@ -136,7 +136,7 @@ endif
 # above.
 $$(call forward-vars,$$(STAGE1_ACI_IMAGE_$1), \
 	ACTOOL STAGE1_ACIDIR_$1)
-$$(STAGE1_ACI_IMAGE_$1): $$(ACTOOL_STAMP) | $$(BINDIR)
+$$(STAGE1_ACI_IMAGE_$1): $$(ACTOOL_STAMP) | $$(TOOLSDIR)
```

Comment:
Uh, this change is wrong, I think. We have an order-only dependency on `$(BINDIR)` here, because this is where we put our generated stage1 images. So we want to be sure that `$(BINDIR)` exists before we run `actool`.